parameters/hardware/sensors/back_laser_filters_parameters.yaml

```diff
@@ -12,6 +12,16 @@ scan_filter_chain:
         type: filters/MultiChannelMedianFilterFloat
         params:
           number_of_observations: 3
+- type: laser_filters/LaserScanBoxFilter
```

Comment:
@reinzor @MatthijsBurgh @MaxBaeten do you know if this is correct? (git blame says Rokus but he has nothing to do with it)

Reply:
Indeed, this is me. I have tuned and tested it on the robot. So as long as we want all laser points within the base to be removed, it is correct.